platform/servicecontrol.include.md

```diff
@@ -1,12 +1,10 @@
-ServiceControl is a suite of back-end tools that collect useful information about a running NServiceBus system. It collects data on every single message flowing through the system (Audit Queue), errors (Error Queue), as well as additional information regarding sagas, endpoints [heartbeats](/monitoring/heartbeats/) and [custom checks](/monitoring/custom-checks/) (Control Queue). The information is then exposed to [ServicePulse](/servicepulse) and [ServiceInsight](/serviceinsight) via an HTTP API and SignalR notifications.
+ServiceControl is a suite of back-end tools that collect useful information about a running NServiceBus system. It collects data on every single message flowing through the system (via an audit queue), errors (via an error queue), as well as additional information regarding sagas, endpoints [heartbeats](/monitoring/heartbeats/) and [custom checks](/monitoring/custom-checks/) (via a control queue). The information is then exposed to [ServicePulse](/servicepulse) and [ServiceInsight](/serviceinsight) through an HTTP API and SignalR notifications.
```

Comment:
> ...through an HTTP API and SignalR notifications.

I'm not sure this is an improvement on the previous wording, but I can see why it's been done due to the addition of "via" in a few other places. I think a better solution is to restructure that other sentence, which reads quite awkwardly. Perhaps:

```suggestion
ServiceControl is a background process that collects useful information about an NServiceBus system. It uses an audit queue to collect every message flowing through the system and an error queue to collect failed messages. It can also collect saga state changes, endpoint [heartbeats](/monitoring/heartbeats/), and perform [custom checks](/monitoring/custom-checks/) using a control queue. This information is exposed to [ServicePulse](/servicepulse) and [ServiceInsight](/serviceinsight) via an HTTP API and SignalR notifications.
```